googletest/test/gtest_prod_test.cc

```diff
@@ -30,8 +30,8 @@
 //
 // Unit test for gtest_prod.h.
 
-#include "production.h"
 #include "gtest/gtest.h"
+#include "test/production.h"
```

Comment:
How is this related to the bazel change? Please revert and just leave the bazel related changes.